en/docs/install-and-setup/setup/distributed-deployment/synchronizing-artifacts-in-a-gateway-cluster.md

```diff
@@ -47,7 +45,12 @@ At startup, the Gateway will look for the APIs with labels that it is subscribed
 
 Gateways are subscribed to the Traffic Manager. There is an extension in the Gateway to get the Synapse artifacts and deploy them in the memory. Gateways can subscribe to multiple labels.
 
-### Configuring the inbuilt Artifact Synchronizer
+### Gateway Labels and Environments
+
+If you need to deploy an API in a specific gateway, you could use either [Gateway Labels]({{base_path}}/learn/api-gateway/maintaining-separate-production-and-sandbox-gateways/#gateway-labels) or [Gateway Environments]({{base_path}}/learn/api-gateway/maintaining-separate-production-and-sandbox-gateways) to expose that particular Gateway. 
```

Comment:
```suggestion
If you need to deploy an API in a specific Gateway, you could use either [Gateway Labels]({{base_path}}/learn/api-gateway/maintaining-separate-production-and-sandbox-gateways/#gateway-labels) or [Gateway Environments]({{base_path}}/learn/api-gateway/maintaining-separate-production-and-sandbox-gateways) to expose that particular Gateway. 
```

Reply:
Let's have all compontent names in title case. For example, Traffic Manager, Gateway, Publisher, etc. Please fix in all instances.

Reply:
Fixed with c71a601